experiments/calibration/calibrate_land.jl

```diff
@@ -1,5 +1,19 @@
+using Dates # needs to be called again for workers
+
+model = "bucket" # or "bucket"
+const variable_list = ["lhf", "shf"] # variables you want to capture by adjusting your priors
+const n_iterations = 10 # 1 iterations takes ~ 1.5 hour with current settings (resolution, 2 year simulation)
+const spinup_period = Year(1)
+const nelements = (101, 15) # resolution - (horizontal elements (lon,lat), vertical elements (soil discretization))
+const start_date = DateTime(2008, 12, 01) # this is the start of the forward model spinup
```

Comment:
any reason why Dec and not Jan?

Reply:
Because the "seasons" we use (DJF, MAM, JJA, SON) start in December and not January